guides/introduction/learning.md

```diff
@@ -1,15 +1,56 @@
 # Learning
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin nibh augue, suscipit a, scelerisque sed, lacinia in, mi. Cras vel lorem. Etiam pellentesque aliquet tellus. Phasellus pharetra nulla ac diam. Quisque semper justo at risus. Donec venenatis, turpis vel hendrerit interdum, dui ligula ultricies purus, sed posuere libero dui id orci. Nam congue, pede vitae dapibus aliquet, elit magna vulputate arcu, vel tempus metus leo non est. Etiam sit amet lectus quis est congue mollis. Phasellus congue lacus eget neque. Phasellus ornare, ante vitae consectetuer consequat, purus sapien ultricies dolor, et mollis pede metus eget nisi. Praesent sodales velit quis augue. Cras suscipit, urna at aliquam rhoncus, urna quam viverra nisi, in interdum massa nibh nec erat.
+Even if Elixir is rather a young programming language, there are a lot of materials to begin and continue learning. Here you can find selected books, articles, online courses and videos with a short descriptions, which will be really helpful while mastering Elixir.
```

Comment:
Even though Elixir is rather a young programming language, there are a lot of materials to start with and continue learning. Below you can find selected books, articles, online courses and videos with short descriptions, that will help you master Elixir.